### ISSUE TITLE
Make the Cupertino Date Picker easier to driver with Flutter driver

### ISSUE THREAD

**Issue by jonsamwell** [flutter/flutter#49680]
Similar to #49100 if would be great to make this widget easier to driver using Flutter driver by adding well known keys to the days, months, years, hours, minutes, seconds CupertinoPicker widgets within this control.  These picker could then be scrolled and tapped and thus any date and/ or time could be picked.

At the moment this widget is impossible to driver using the driver.

I'm happy to do a PR for this so let me know any thoughts or if I have missed something simple with driving this element with the driver.

**Comment by xster** [comment]
What API are you proposing on the driver?

**Comment by jonsamwell** [comment]
@xster I don't think this need a custom api exposed on the driver.  It would be better to have keys on certain elements within the Cupertino date picker so that they can be controlled via the driver
